mbof/models.py

```diff
@@ -144,9 +144,10 @@ class Intention(models.Model):
     INTENTION_DECLINED = 'declined' #may use in future
     event = models.ForeignKey(Event)
     respondent = models.ForeignKey(User, default=currentUserObject, editable=False)
-    intention = models.CharField(max_length=5, choices=(
+    intention = models.CharField(max_length=8, choices=(
```

Comment:
Does `max_length` need to be set?  What happens if you don't specify it?

If you do set `max_length`, set the argument programmatically by finding which of the choices has the longest value and use the number of characters in that value.